.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+*Note: for `ggml`-related changes (including to the CMake build system), please open a PR in https://github.com/ggml-org/llama.cpp. Doing so will make your PR more visible and more likely to be reviewed.*
```

Comment:
The wording of "ggml-related changes" could be more clear, everything in this repository can be understood as being related to ggml. Maybe something like "changes to the ggml core library"?